CMakeLists.txt

```diff
@@ -197,16 +197,16 @@ endif()
 
 # Parallelproj
 if(NOT DISABLE_Parallelproj_PROJECTOR)
-    find_package(parallelproj 1.0 CONFIG)
+    find_package(parallelproj CONFIG)
```

Comment:
I guess still best to keep 1.0 here. why delete it? If anything, I'd put 1.0.1 here.

Reply:
A misread of CMake find_package documentation. Fixed (I think)